### PR TITLE
Update README & Ln.java comment

### PR DESCRIPTION
**Pull request by e1e0**
- `###` Summary
- `##` Custom paths
...

#3063

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -174,6 +148,24 @@ It accepts command-line arguments, listed by:
 scrcpy --help
 ```
 
+
+## Custom paths
+
+(Linux): For Android Studio (SDK) `adb` to be `adb` command, do `sudo ln --symbolic ### /usr/local/bin/adb`.
```
Comment (e1e0) [inline README.md:154]: #3063 .

File: README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ _pronounced "**scr**een **c**o**py**"_
 [Read in another language](#translations)
 
 This application provides display and control of Android devices connected via
-USB (or [over TCP/IP](#tcpip-wireless)). It does not require any _root_ access.
+USB (or [over TCP/IP](#tcpip-wireless)). It _does not require_ any root access.
```
Comment (rom1v) [inline README.md:10]: Why?

File: README.md [hunk 3]
```diff
@@ -56,52 +56,44 @@ control it using keyboard and mouse.
...
 
 [Ebuild]: https://wiki.gentoo.org/wiki/Ebuild
-[ebuild-link]: https://github.com/maggu2810/maggu2810-overlay/tree/master/app-mobilephone/scrcpy
-
-You could also [build the app manually][BUILD] ([simplified
-process][BUILD_simple]).
 
 
-### Windows
+**Windows > archive**
```
Comment (rom1v) [inline README.md:96]: I think I prefer the current version, with 1 section for linux, 1 section for Windows and 1 section for macOS.

File: README.md (hunk 1 above)
Comment (rom1v) [reply README.md:154]: I'm not sure this is the right place to explain this (it is independent of scrcpy).

Either you use the default `adb` (wherever it comes from in your PATH), either you can overwrite it using the `ADB` environment variable.

File: README.md [hunk 4]
```diff
@@ -1105,10 +1086,7 @@ See [BUILD].
 
 ## Common issues
 
-See the [FAQ].md).
```
Comment (rom1v) [inline README.md:1108]: Good catch :+1:

ef24690eee805b447ff1d2092298bbb99777a58f (not merged yet)

File: README.md (hunk 2 above)
Comment (e1e0) [reply README.md:10]: Oh, okay. 🤔

File: README.md [hunk 5]
```diff
@@ -56,123 +56,105 @@ control it using keyboard and mouse.
...
 
-Build from sources: [BUILD] ([simplified process][BUILD_simple])
-
-[BUILD]: BUILD.md
-[BUILD_simple]: BUILD.md#simple
-
-
-### Linux
-
-On Debian and Ubuntu:
```
Comment (e1e0) [inline README.md:71]: (line) 59

File: README.md [hunk 6]
```diff
@@ -56,123 +56,105 @@ control it using keyboard and mouse.
...
 
-It's also available in [MacPorts], which sets up adb for you:
+It accepts command-line arguments, listed by:
 
 ```bash
-sudo port install scrcpy
+scrcpy --help
 ```
 
-[MacPorts]: https://www.macports.org/
```
Comment (e1e0) [inline README.md:157]: [114](#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R114)

File: README.md [hunk 7]
```diff
@@ -56,123 +56,105 @@ control it using keyboard and mouse.
...
+- Fedora / [COPR] package, available at [`scrcpy`](https://copr.fedorainfracloud.org/coprs/zeno/scrcpy/)
 
-For Fedora, a [COPR] package is available: [`scrcpy`][copr-link].
+- Gentoo / [Ebuild], available at [`scrcpy/`](https://github.com/maggu2810/maggu2810-overlay/tree/master/app-mobilephone/scrcpy).
 
 [COPR]: https://fedoraproject.org/wiki/Category:Copr
-[copr-link]: https://copr.fedorainfracloud.org/coprs/zeno/scrcpy/
-
-
-For Gentoo, an [Ebuild] is available: [`scrcpy/`][ebuild-link].
```
Comment (e1e0) [inline README.md:95]: - 80

File: README.md [hunk 8]
```diff
@@ -56,123 +56,105 @@ control it using keyboard and mouse.
...
 
 [Ebuild]: https://wiki.gentoo.org/wiki/Ebuild
-[ebuild-link]: https://github.com/maggu2810/maggu2810-overlay/tree/master/app-mobilephone/scrcpy
 
-You could also [build the app manually][BUILD] ([simplified
-process][BUILD_simple]).
```
Comment (e1e0) [inline README.md:101]: - 124

File: README.md [hunk 9]
```diff
@@ -56,123 +56,105 @@ control it using keyboard and mouse.
...
+You could build manually from sources: [BUILD.md](BUILD.md), [Simplified process](BUILD.md#simple).
 
-```bash
-brew install scrcpy
-```
 
-You need `adb`, accessible from your `PATH`. If you don't have it yet:
+## Run
+
+Plug in an Android device, **connect with [`adb`](https://developer.android.com/studio/command-line/adb)** and execute:
```
Comment (e1e0) [inline README.md:129]: - 165

File: README.md [hunk 10]
```diff
@@ -56,123 +56,105 @@ control it using keyboard and mouse.
...
 
 ```bash
 scoop install scrcpy
 scoop install adb    # if you don't have it yet
 ```
 
-[Scoop]: https://scoop.sh
 
-You can also [build the app manually][BUILD].
+### macOS, in [MacPorts](https://www.macports.org/)
```
Comment (e1e0) [inline README.md:114]: 151 **↓**

**Review by e1e0 — COMMENTED** [review]
Is this good? Do more? @rom1v

File: README.md [hunk 11]
```diff
@@ -56,113 +56,79 @@ control it using keyboard and mouse.
...
-### macOS
-
-The application is available in [Homebrew]. Just install it:
-
-[Homebrew]: https://brew.sh/
-
-```bash
-brew install scrcpy
-```
-
```
Comment (e1e0) [inline README.md:144]: -> 61

File: README.md [hunk 12]
```diff
@@ -56,113 +56,79 @@ control it using keyboard and mouse.
...
 ```bash
 sudo port install scrcpy
 ```
 
-[MacPorts]: https://www.macports.org/
+### Manual build
 
-
-You can also [build the app manually][BUILD].
+You could build manually from sources: [BUILD.md](BUILD.md), [Simplified process](BUILD.md#simple).
```
Comment (e1e0) [inline README.md:125]: **63 - 66**

**Pull request closed without merging**